### PR TITLE
[quant][graphmode] Add e2e test for conv+bn

### PR DESCRIPTION
**Pull request by jerryzh168**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #27403 [quant][graphmode] Minor fix and turn off fold_convbn
* **#27348 [quant][graphmode] Add e2e test for conv+bn**

Summary:
att

Test Plan:
python test/test_quantization.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18182920](https://our.internmc.facebook.com/intern/diff/D18182920)

### CODE REVIEW

File: test/test_quantization.py [hunk 1]
```diff
@@ -699,6 +698,34 @@ def test_single_layer(self):
...
+            '': default_qconfig
+        }
+        model_script = quantize_script(
+            torch.jit.script(conv_model_to_script),
+            qconfig_dict,
+            default_eval_fn,
+            [self.img_data],
+            inplace=False)
+        result_eager = model_eager(self.img_data[0][0])
+        result_script = get_forward(model_script._c)(self.img_data[0][0])
```
Comment (ZolotukhinM) [inline test/test_quantization.py]: You could call it the same way you call the eager model:
```
result_script = model_script(self.img_data[0][0])
```

File: test/test_quantization.py [hunk 2]
```diff
@@ -699,6 +698,34 @@ def test_single_layer(self):
...
+        # copy the weight from eager mode so that we can
+        # compare the result of the two quantized models later
+        conv_model_to_script.conv.weight = torch.nn.Parameter(conv_model.conv.weight.detach())
+        fuse_modules(conv_model, ['conv', 'bn'], inplace=True)
+        model_eager = quantize(conv_model, default_eval_fn,
+                               self.img_data)
+        qconfig_dict = {
+            '': default_qconfig
+        }
+        model_script = quantize_script(
```
Comment (ZolotukhinM) [inline test/test_quantization.py]: We need to somehow verify that we're actually running quantized ops. If for some reason fusion doesn't work, this test won't catch it.

**Review by ZolotukhinM — APPROVED** [review]
This test would not catch if fusion (or any of other "optional" passes) would stop working, but it's still better to have it. Please think about how we can test passes that can silently do nothing.

File: test/test_quantization.py (hunk 2 above)
Comment (jerryzh168) [reply test/test_quantization.py]: do you have similar tests for graph executor?

**Pull request closed without merging**